### ISSUE TITLE
Live streaming quality

### ISSUE THREAD

**Issue by lordmen99** [angelOnly/LiveStreaming#1]
hi there, i see u use tencent live sdk. its good for live streaming?  i mean smooth or get lag?

**Comment by angelOnly** [comment]
I just learning it for improve my coding level，but I am a  novice，so I don't known any more. But I have used the tencent demo named xiaozhibo, I will give you the apk and url address, you can download it to test. I think it is smooth, but I can't give you the certain answer. I'm so sorry to can't help you this issue.I will try my best to help you.If you have any question about it. please issue you question, I can help you.
 the xiaozhibo url is https://www.qcloud.com/document/product/454/6554
the suixinbo url is https://github.com/zhaoyang21cn/iLiveSDK_Android_Suixinbo
in these articles  I have given you, there are demo and resource code, you can known more about tencent live sdk.

**Comment by lordmen99** [comment]
ok, thx u. where i can download demo apk? cause my internet provider block to tencent web

**Comment by angelOnly** [comment]
can't you enter the tencent web?

**Comment by angelOnly** [comment]
I can send a email to you.can you leave you email there? I will send you after I download them.

**Comment by lordmen99** [comment]
email me dnang99@gmail.com. btw, u join this course http://www.cniao5.com/course/10121 ?

**Comment by angelOnly** [comment]
Yes,I learn it at this course. do you have any question? I will send you later.

**Comment by angelOnly** [comment]
can you known Chinese?

**Comment by angelOnly** [comment]
have you received the email?

**Comment by lordmen99** [comment]
I use google translator. Actualy how much cost use live broadcast from
Tencent SDK?

**Comment by lordmen99** [comment]
source from that course its same with u sent to me in my email?

**Comment by angelOnly** [comment]
no, which I send to you is the demo of tencent, If you want to my source, I can give you. but I think if you want to set up your background server, It may cost you some money. you know it isn't me to set up the background server. so I don't know the details about it. you can ask tencent customer server for help.

**Comment by lordmen99** [comment]
i can do setup for backend? what type of backend? nodejs or just php backend?. please send , i will do setup for u  too. we learn do together

**Comment by angelOnly** [comment]
I don't know how to setup the backend, but I can send you the source, I think the nodejs or php can setup the backend too. you can try it.if you setup the backend, please tell me how to setup. I also want to learn this.Thank you.

**Comment by angelOnly** [comment]
where are you from?

**Comment by lordmen99** [comment]
ok, i will send guide when i success and working to setup the whole app. cause i have too source from agora.io but its still get bug. from indonesia

**Comment by angelOnly** [comment]
oh,I am from China

**Comment by lordmen99** [comment]
ok, send me source now. i will take look and try setup it

**Comment by angelOnly** [comment]
Ok, wait a moment

**Comment by lordmen99** [comment]
ok

**Comment by angelOnly** [comment]
I will have a rest now, if you have any question, send me email, I will reply you when I see.

**Comment by lordmen99** [comment]
btw, what function app from that course? i see its have some  = 
1. live streaming
2. counter user
3. send gift and point
4. purchase and withdraw point already too?

**Comment by angelOnly** [comment]
yes, can you enter the http://www.cniao5.com/course/10121 ? just as you see.

**Comment by lordmen99** [comment]
where is backend? i just see json file and android source, and pptx file

**Comment by angelOnly** [comment]
you can't see the backend, I also don't have, this is the product of cniaowo company, they won't give you and me, so there is only the android source, I can't help you, if I have the backend, I think I also can setup the backend. if you want to test the apk, you must register the account in cniaowo.  this is the register url. http://www.cniao5.com/auth/reg.html .  this is all that I can help you, nothing else.

**Comment by lordmen99** [comment]
ok, i see they use api cniao. ok i will try build apk and test it

**Comment by angelOnly** [comment]
ok

**Comment by lordmen99** [comment]
and guide from that course in video or pdf?

**Comment by angelOnly** [comment]
I don't have pdf, you can guide from my articles. if you want watch video, you must buy it, but it is the Chinese teacher, I don't know whether do you know Chinese, If you can't understand Chinese, I suggest you not to buy.

**Comment by lordmen99** [comment]
i see, i can understand with video. but how size from each video course.. ? if uhave any demo apk ready with live streaming. cause i have done build in android studio,its succes register but when login success still cant enter live streaming

**Comment by angelOnly** [comment]
you can't enter live Streaming? but i can

**Comment by lordmen99** [comment]
i use register and login username. u use default android source with same u sent to me?

**Comment by angelOnly** [comment]
Is it your company to ask you develop the Live App? I can enter the live streaming.  The code I send to you is final code of the course. it is the same as I use.

**Comment by angelOnly** [comment]
can you push a live?

**Comment by angelOnly** [comment]
I am pushing a live. can you enter?

**Comment by lordmen99** [comment]
i still cant login in live streaming page

**Comment by angelOnly** [comment]
do you have wechat?

**Comment by lordmen99** [comment]
yes

**Comment by lordmen99** [comment]
ok can login now.. :D

**Comment by angelOnly** [comment]
you can leave you wechat.

**Comment by lordmen99** [comment]
I TRY stream, but still cant load, its must add cover image?. its my wechat = Danangs

**Comment by angelOnly** [comment]
yes,I can stream

**Comment by lordmen99** [comment]
try again.., i will join ur room

**Comment by angelOnly** [comment]
ok

**Comment by angelOnly** [comment]
I have add you in wechat

**Comment by lordmen99** [comment]
try ping me in wechat

**Comment by angelOnly** [comment]
？I don't understand you

**Comment by lordmen99** [comment]
i mean what ur wechat id?

**Comment by angelOnly** [comment]
a18581391726

**Comment by lordmen99** [comment]
i still cant do stream. when click start live page back again

**Comment by lordmen99** [comment]
i mean this http://cniao5-imgs.qiniudn.com/o_1b927fk2b1e1u19h1o31ijk6n2f.jpg, u see 10000 its balance user, where when user send gift, the balance will decrease as value of gift

**Comment by angelOnly** [comment]
i don't know

**Comment by lordmen99** [comment]
i think its stilll get lag.  better use agora.io

**Comment by lordmen99** [comment]
hi, i still cant make live streaming after login.. why this.
